### PR TITLE
Added Confidence Threshold value to Config

### LINKED ISSUE #54: ObjectRec:  Make confidence threshold a config item
Add ability to set the threshold.  We can continue, if not set, to default to .5.  But allow set to 0 (meaning report all).  

Optional:  If threshold is > 1, divide by 100 to turn into a percentage.  I.e. if I set it to 50, than means 50%.

### PR DESCRIPTION
**Pull request by namirfa**
Fixes #54 - Threshold can now be specified using the config. If no threshold is specified then the default value of 0.5 is used. 

Config values can be between 0-1 or between 0-100, anything outside of these ranges will prompt a warning that the value was invalid, and the threshold will be set to 0.5.

### CODE REVIEW

File: objectRecognitionSource/src/main/java/edu/ml/tensorflow/classifier/YOLOClassifier.java [hunk 1]
```diff
@@ -18,20 +18,22 @@
  * I also used this class in my android sample application here: https://github.com/szaza/android-yolo-v2
  */
 public class YOLOClassifier {
-    private final static float OVERLAP_THRESHOLD = 0.5f;
+    private static float OVERLAP_THRESHOLD;
```
[the remark this answers is not in the document]
Comment (namirfa) [reply objectRecognitionSource/src/main/java/edu/ml/tensorflow/classifier/YOLOClassifier.java]: To be completely honest, I am not exactly sure but from what I can understand there is one threshold being used to check agains the confidence of the "recognitions" (THRESHOLD), and then another one being used to check against the overlap of the boxes being produced (OVERLAP_THRESHOLD). This portion of the code was taken from the project listed in the initial spec that you sent us.

**Comment by namirfa** [comment]
Let me know if I properly addressed the issues you mentioned! I do not have much experience writing tests with JUnit so let me know if you see any issues.

**Comment by fhcarter** [comment]
It will be a good idea to hand test as well -- verify that we see lower/higher confidence for identified objects.  The test verifies that we process it, but we should also make certain it has an effect.

**Comment by fhcarter** [comment]
Once verified, also need to update the readme to document it :-)

**Pull request merged**